code/web/src/modules/user/ProfileForm.js

```diff
@@ -28,14 +28,38 @@ class ProfileForm extends Component {
       userDetails: {
         id: 0,
         image: '',
-        email: this.props.user.details.email,
-        shippingAddress: this.props.user.details.address || '',
-        description: this.props.user.details.description || ''
+        email: '',
+        shippingAddress: '',
+        description: ''
       },
       isLoading: false
     }
   }
 
+  componentDidMount() {
+    let shippingAddress = '';
+    let description = '';
+    let userDetails; 
+
+    if (this.props.user.details.address) {
```

Comment:
so grateful for you Kara, and for finding that the store is already in fact updated by this point!! Very much appreciate getting to work with you! :)